packages/sdk/src/actions/waitForFillTx.ts

```diff
@@ -1,11 +1,15 @@
-import { Address, Hash, Hex, parseEventLogs } from "viem";
+import { Address, Hash, Hex, Log, parseEventLogs } from "viem";
 import { ConfiguredPublicClient } from "../types/index.js";
-import { spokePoolAbi } from "../abis/SpokePool.js";
+import { spokePoolAbiV3, spokePoolAbiV4 } from "../abis/SpokePool/index.js";
 import { FillStatus, waitForFillByDepositTx } from "./getFillByDepositTx.js";
-import { LoggerT, MulticallHandlerAbi } from "../utils/index.js";
+import {
+  bytes32ToAddress,
+  LoggerT,
+  MulticallHandlerAbi,
+} from "../utils/index.js";
 
 export type WaitForFillTxParams = {
-  depositId: number;
+  depositId: bigint;
```

Comment:
So this is a breaking change right? Maybe it makes sense to allow `number  | bigint` here and convert to a bigint in the function logic